### ISSUE TITLE
Button aria attributes

### ISSUE THREAD

**Issue by swese44** [Microsoft/YamUI#8]
Button's aria-labelledby attribute has a number value, should be removed entirely? Maybe aria-disabled isn't necessary if disabled is already present?
